lib/rules/name-property-required.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Require a name property in Vue components
+ * @author LukeeeeBennett
+ */
+'use strict'
+
+const utils = require('../utils')
+
+function isNameProperty (node) {
+  return node.type === 'Property' && node.key.name === 'name'
```

Comment:
Please check if it is computed property.

Reply:
It now ignores computed properties, is this what you wanted?